concepts/score.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title : Score
+---
+
+Scores are a crucial part of developing your LLM application or agent.
+
+They allow your users to manually evaluate their interactions with your application, which can be used
+to improve the performance and accuracy of your system.
+
+Scores may also be generated by AI via automatic evaluations. 
+
+<Frame caption="Example of Scores collected on Literal">     
+  <img
+    src="/images/scores.png"
+    alt="A list of scores left by users"
+  />
+</Frame>
+
+Scores are tied to a Step or a Generation. As such, they appear in their respective info panels:
+
+<Frame caption="Example of Scores on a Step">
+  <img
+    src="/images/scores-on-step.png"
+    alt="A list of scores left by users on a step"
+  />
+</Frame>
+
+### Create a score
+
+#### From the application
+
+The Literal application offers an easy way to manage scores: Score Templates. 
+
+To create a Score Template, click on "Create score" and select the "Create Template" option:
+
+<Frame caption="Create Template option">
+  <img
+    src="/images/create-score-template.png"
+    alt="The option to create a template"
+  />
+</Frame>
+
+Via Score Templates, admin users can control and expose to users the various types of evaluations
+allowed from the application. Score Templates come in two flavors: *Categorical* and *Continuous*. 
+
+*Categorical* templates let you create a set of categories, each tied to a numeric value:
+
+<Frame caption="Create categorical Score Template">
+  <img
+    src="/images/create-categorical-template.png"
+    alt="Categorical template form"
+  />
+</Frame>
+
+*Continuous* templates offer a minimum and a maximum value which users can then select from to score.
+
+<Frame caption="Create continuous Score Template">
+  <img
+    src="/images/create-continuous-template.png"
+    alt="Continuous template form"
+  />
+</Frame>
+
+From a Step or a Generation, admins can then score by selecting a template and filling
+in the required form fields.
+
+Scores can be deleted individually from a Step or a Generation via their delete button.
+
+By default, any admin user may create a score template. Deletion is left in the hands of
+project owners, from the Settings page.
+
+#### Programmatically
+
+The SDKs provide score creation APIs with all fields exposed. Scores must be tied either
+to a Step or a Generation object.
+
+<CodeGroup>
+```python Python
+score = await client.api.create_score(
```

Comment:
I think it would be great to add an example code on how to get the step_id or generation_id, such that the user sees the two stages: created a step, then added a score to it